streams/src/test/java/org/apache/kafka/streams/processor/internals/TestToInternal.java

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
nit: could we rename to sth. similar to `InternalTopologyAccessor` as we did in the other PR?